mono/metadata/Makefile.am

```diff
@@ -20,7 +20,25 @@ win32_sources = \
 	w32process-win32-internals.h \
 	w32socket-win32.c \
 	w32error-win32.c \
-	w32subset.h
+	w32subset.h \
```

Comment:
This might get compiled twice, Boehm and sgen, due to how it is structured.
That could be easily fixed.